backend/src/main/java/com/bakdata/conquery/models/config/SqlConnectorConfig.java

```diff
@@ -11,22 +16,21 @@
 @AllArgsConstructor
 public class SqlConnectorConfig {
 
-	public static final String DEFAULT_PRIMARY_COLUMN = "pid";
-
 	boolean enabled;
 
-	private Dialect dialect;
-
 	/**
 	 * Determines if generated SQL should be formatted.
 	 */
 	private boolean withPrettyPrinting;
 
-	private String databaseUsername;
-
-	private String databasePassword;
+	/**
+	 * Keys must match the name of existing {@link Dataset}s.
+	 */
+	@Getter(AccessLevel.PRIVATE)
+	private Map<String, DatabaseConfig> databaseConfigs;
 
-	private String jdbcConnectionUrl;
+	public DatabaseConfig getDatabaseConfig(Dataset dataset) {
+		return databaseConfigs.get(dataset.getName());
+	}
```

Comment:
Ich hatte es etwas anders in Gedanken, aber so passt es erst mal.

Meine Idee ging eher dahin, dass die datasetId in den jdbc connection String getemplated wird

Reply:
Alles klar, ich notiere mir das mal dass wir es zu gegebener Zeit nicht aus den Augen verlieren.